src/rules/index.d.ts

```diff
@@ -21,6 +21,7 @@ import type { VitestRules } from './vitest';
 import type { VueRules } from './vue';
 import type { VueI18nRules } from './vue-i18n';
 import type { VuePugRules } from './vue-pug';
+import type { YmlRules } from './yml';
```

Comment:
How did you found out that you need to do that change manually?
I'm proud of you
Or is this also already in the automation ^^

Reply:
I don't believe it's a part of automation, I actually started with manual work and then realized that ~97% of it is automated ;D